src/guide/diagrams-src/ffi.tex

```diff
@@ -0,0 +1,73 @@
+\documentclass[tikz,border={0 1}]{standalone}
```

Comment:
There is a [`markdown-it-texmath`](https://www.npmjs.com/package/markdown-it-texmath) package that could help with TeX rendering directly from source, but... I tried it's [demo](https://goessner.github.io/markdown-it-texmath/index.html) and didn't find a way how to display this file.

Reply:
thanks for the info! if we have more diagrams, we might want to look into it to automate the rendering